### PR TITLE
Fresh batch of data. 

### PR DESCRIPTION
**Pull request by geramirez**
Missing data that needs to be resolved. 
### Missing Offices
- Office of Natural Resources Revenue - DOI
- U.S. Joint Forces Command - DOD 
- Office of Acquisition & Logistics - VA
### Missing Address
- HUD
- MSPB

### CODE REVIEW

**Comment by konklone** [comment]
5902776_t0lgoixj_c

**Comment by aarongraves** [comment]
Can I ask where you are getting the contact information for the DoD and how we go about getting it changed (at least changed here to start)?

Thanks,

Aaron Graves
OSD/JS FOIA Office
(571) 372-0435

-----Original Message-----

**Comment by geramirez** [comment]
> Can I ask where you are getting the contact information for the DoD and how we go about getting it changed (at least changed here to start)?

Hey Aaron, 

For now, the fastest way to update your data is through Jake at the DOJ because we are still scraping foia.gov for our main contact data. 

In a couple weeks we should have a contact updater ready and we would really appreciate if you could test it. If you're interested please let us know. While we work you can take a look at the contact updater designs that we are building [here](https://github.com/18F/foia-hub/issues/723).

File: contacts/data/DOI.yaml [hunk 1]
```diff
@@ -1114,86 +1226,12 @@ departments:
   website: http://www.doi.gov/oig/foia/index.cfm
 - address:
     address_lines:
```
Comment (geramirez) [inline contacts/data/DOI.yaml:1228]: disappeared

File: contacts/data/DOI.yaml [hunk 2]
```diff
@@ -1419,22 +1477,30 @@ departments:
...
   service_center:
     phone:
-    - 202-208-2909
+    - 202-208-5840
   top_level: false
-  website: http://www.osmre.gov/contacts/Foia.shtm
+  website: http://www.osmre.gov/lrg/foia.shtm
 - address:
     address_lines:
     - Veronica Herkshan
```
Comment (ramirezg) [inline contacts/data/DOI.yaml]: Why is this gone

File: contacts/data/DoD.yaml [hunk 3]
```diff
@@ -160,100 +168,20 @@ departments:
...
+  phone: 703-693-2634
   public_liaison:
     name: Anh Trinh
     phone:
-    - 703-614-8500
+    - 703-693-2634
   reading_rooms:
   - - FOIA LIBRARY
     - http://www.foia.af.mil/foialibrary/index.asp
   request_form: https://www.efoia.af.mil/palMain.aspx
```
Comment (ramirezg) [inline contacts/data/DoD.yaml]: stats gone

File: contacts/data/DoD.yaml [hunk 4]
```diff
@@ -333,83 +254,18 @@ departments:
     - 703-428-6462
   reading_rooms:
   - - Reading Room
-    - https://www.foia.army.mil/ReadingRoom/landing.aspx
```
Comment (ramirezg) [inline contacts/data/DoD.yaml]: stats gone

File: contacts/data/DoD.yaml [hunk 5]
```diff
@@ -459,127 +315,37 @@ departments:
...
+  name: Department of the Navy
   phone: 202-685-0412
   public_liaison:
-    name: Steve Muck
+    name: Christopher Julka
     phone:
-    - 202-695-1297
+    - 703-697-0031
   request_form: https://foia.navy.mil/foia/webbas02.nsf/(vwwebpage)/home.htm?opendocument
-  request_time_stats:
```
Comment (ramirezg) [inline contacts/data/DoD.yaml]: stats gone

File: contacts/data/DoD.yaml [hunk 6]
```diff
@@ -1257,79 +1068,6 @@ departments:
   reading_rooms:
   - - Electronic Reading Room
     - http://www.dla.mil/foia-privacy/Pages/FOIAReading.aspx
-  request_time_stats:
```
Comment (ramirezg) [inline contacts/data/DoD.yaml]: Stats gone

File: contacts/data/DoD.yaml [hunk 7]
```diff
@@ -2156,111 +1973,84 @@ departments:
...
-    name: Jim Hogan
+    name: Linda Thomas
     phone:
-    - 571-372-0462
-  reading_rooms:
-  - - FOIA Electronic Reading Room
-    - http://www.tricare.mil/tma/privacy/FOIAeletric.aspx
+    - 703-681-7500
   request_form: http://www.tricare.mil/tma/privacy/FOIARequestForm.aspx
   request_time_stats:
```
Comment (ramirezg) [inline contacts/data/DoD.yaml]: Stats gone

File: contacts/data/DoD.yaml [hunk 8]
```diff
@@ -2555,106 +2373,26 @@ departments:
   website: http://www.eucom.mil/policies-and-compliance/freedom-of-information-act-foia-requestor-service-center
 - address:
     address_lines:
+    - Kristina Roth
     - FOIA Contact
```
Comment (geramirez) [inline contacts/data/DoD.yaml:2784]: Office gone

File: contacts/data/HUD.yaml [hunk 9]
```diff
@@ -1,13 +1,4 @@
 abbreviation: HUD
-address:
```
Comment (geramirez) [inline contacts/data/HUD.yaml:2]: Address gone

File: contacts/data/MSPB.yaml [hunk 10]
```diff
@@ -1,16 +1,8 @@
 abbreviation: MSPB
```
Comment (geramirez) [inline contacts/data/MSPB.yaml:1]: Address gone

File: contacts/data/NARA.yaml [hunk 11]
```diff
@@ -103,32 +103,7 @@ departments:
   website: http://www.archives.gov/foia/index.html
 - address:
     address_lines:
```
Comment (ramirezg) [inline contacts/data/NARA.yaml]: Office gone

File: contacts/data/NASA.yaml [hunk 12]
```diff
@@ -174,95 +189,30 @@ departments:
...
   phone: 661-276-2704
   public_liaison:
     name: Kim Lewis-Bias
     phone:
     - 661-276-2704
-  reading_rooms:
-  - - Electronic Reading Room
-    - http://www.nasa.gov/centers/armstrong/about/FOIA/readroom.html
-  request_form: http://socialforms.nasa.gov/foia
   request_time_stats:
```
Comment (ramirezg) [inline contacts/data/NASA.yaml]: stats gone

File: contacts/data/USDA.yaml [hunk 13]
```diff
@@ -1,18 +1,9 @@
 abbreviation: USDA
```
Comment (geramirez) [inline contacts/data/USDA.yaml:1]: Address gone

File: contacts/data/USDA.yaml [hunk 14]
```diff
@@ -325,66 +324,44 @@ departments:
...
+  fax: 202-690-0068
+  name: Departmental Management (OSEC/OCIO/ FOIA Service Center)
+  phone: 202-690-3318
   public_liaison:
-    name: Phyllis Holmes
     phone:
     - 202-720-0068
   reading_rooms:
   - - USDA Agencies' FOIA Reading Rooms
     - http://www.dm.usda.gov/foia/agencyfoia.htm
```
Comment (geramirez) [inline contacts/data/USDA.yaml:364]: Stats gone

File: contacts/data/USDA.yaml [hunk 15]
```diff
@@ -1602,55 +1690,24 @@ departments:
...
+  fax: 202-690-0068
+  name: Office of the Chief Information Officer (OSEC/OCIO/ FOIA Service Center)
+  phone: 202-690-3318
   public_liaison:
-    name: Richard Ciampa
     phone:
     - 202-720-6355
   reading_rooms:
   - - USDA Agencies' FOIA Reading Rooms
     - http://www.dm.usda.gov/foia/agencyfoia.htm
```
Comment (ramirezg) [inline contacts/data/USDA.yaml]: Stats gone

File: contacts/data/VA.yaml [hunk 16]
```diff
@@ -339,68 +399,20 @@ departments:
...
-  name: Office of Assistant Secretary for Policy and Intergovernmental Affairs
+  name: Office of Assistant Secretary for Public and Intergovernmental Affairs
   phone: 202-461-7442
   public_liaison:
-    name: Inez Sidbury
+    name: James Horan
     phone:
     - 202-461-7442
   reading_rooms:
   - - FOIA Library
```
Comment (ramirezg) [inline contacts/data/VA.yaml]: Stats gone

File: contacts/data/VA.yaml [hunk 17]
```diff
@@ -1427,66 +1537,21 @@ departments:
...
+  - Government procurement
+  name: Office of Procurement Policy Services
   phone: 202-632-5286
   public_liaison:
-    name: Richard Ha
-    phone:
-    - 202-632-5286
+    name: James Horan
   reading_rooms:
   - - FOIA Library
```
Comment (geramirez) [inline contacts/data/VA.yaml:1604]: Missing Office Stats gone

**Comment by khandelwal** [comment]
So, what's going on with the missing pieces of information here? Have you had a chance to chase all this down?

**Comment by geramirez** [comment]
@khandelwal Yes, the only pieces of missing information are listed in the initial comment. I'm going to open up a ticket to follow up with the DOJ.

**Pull request merged**